contracts/GNUmakefile

```diff
@@ -0,0 +1,80 @@
+# ALL_FOUNDRY_PRODUCTS contains a list of all products that have a foundry
+# profile defined. Adding a product to this list will make it available for
+# snapshotting.
+ALL_FOUNDRY_PRODUCTS = vrf automation llo functions automation-dev shared
```

Comment:
automation-dev is required because automation has Foundry code in the `dev` folder. This should be moved into the automation folder at some point.

shared is already here even though there are no Foundry tests yet, this will soon change so I already added it